tracing/opencensus_test.go

```diff
@@ -47,3 +48,43 @@ func TestOpenCensusTracerGlobalLifecycle(t *testing.T) {
 	}
 	otherOCT.Finish()
 }
+
+func TestOpenCensusTrace_ApplyConfig_FailingConfigOption(t *testing.T) {
```

Comment:
Remove `_` to be consistent with the rest of the tests in the file.

Reply:
Done.